common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -344,6 +370,22 @@ innovation_night_elf_aerial = {
 	flag = global_maa
 }
 
+innovation_nerubian_tactics = {
```

Comment:
Why siege progress is so big?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/155878208-7cdbd802-42c4-4f76-aaec-f68f53fdd4a7.png)

</details>

Reply:
Just to make them unique, in return they are have less effectiveness with forts above level 10